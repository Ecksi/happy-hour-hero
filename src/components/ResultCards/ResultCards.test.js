import React from 'react';
import { shallow } from 'enzyme';
import ResultsCards from './ResultCards';

describe('ResultsCard', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<ResultsCards />);
  });

  it('matches the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});