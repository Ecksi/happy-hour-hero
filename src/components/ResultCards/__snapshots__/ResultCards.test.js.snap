// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResultsCard matches the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ResultCards
    day="Today"
    drinkSpecials={Array []}
    filteredRestaurants={Array []}
    foodSpecials={Array []}
    happyHours={Array []}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": undefined,
      "className": "resultCardsContainer",
    },
    "ref": null,
    "rendered": null,
    "type": "section",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": undefined,
        "className": "resultCardsContainer",
      },
      "ref": null,
      "rendered": null,
      "type": "section",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
